Source/WebCore/platform/graphics/cpu/arm/filters/FEBlendNeonApplier.cpp

```diff
@@ -176,19 +176,19 @@ bool FEBlendNeonApplier::apply(const Filter&, const FilterImageVector& inputs, F
     if (!destinationPixelBuffer)
         return false;
 
-    auto* destinationPixelArray = destinationPixelBuffer->bytes();
+    auto* destinationPixelArray = destinationPixelBuffer->bytes().data();
 
     auto effectADrawingRect = result.absoluteImageRectRelativeTo(input);
     auto sourcePixelArrayA = input.getPixelBuffer(AlphaPremultiplication::Premultiplied, effectADrawingRect);
 
     auto effectBDrawingRect = result.absoluteImageRectRelativeTo(input2);
     auto sourcePixelArrayB = input2.getPixelBuffer(AlphaPremultiplication::Premultiplied, effectBDrawingRect);
 
-    unsigned sourcePixelArrayLength = sourcePixelArrayA->sizeInBytes();
-    ASSERT(sourcePixelArrayLength == sourcePixelArrayB->sizeInBytes());
+    unsigned sourcePixelArrayLength = sourcePixelArrayA->bytes().size();
+    ASSERT(sourcePixelArrayLength == sourcePixelArrayB->bytes().size());
 
     if (sourcePixelArrayLength >= 8) {
-        applyPlatform(sourcePixelArrayA->bytes(), sourcePixelArrayB->bytes(), destinationPixelArray, sourcePixelArrayLength);
+        applyPlatform(sourcePixelArrayA->bytes().data(), sourcePixelArrayB->bytes().data(), destinationPixelArray, sourcePixelArrayLength);
```

Comment:
This should be fine, because `sourcePixelArrayA->bytes()` remains valid until the call to applyPlatform() finishes. (I admit I had to [look this up](https://stackoverflow.com/a/2506800/1120203) to be certain.)